eventmesh-dashboard-console/pom.xml

```diff
@@ -36,46 +36,30 @@
     </properties>
 
     <dependencies>
-        <!-- Spring Boot Starter -->
-        <dependency>
-            <groupId>org.springframework.boot</groupId>
-            <artifactId>spring-boot-starter-web</artifactId>
-        </dependency>
-
         <!-- EventMesh Dashboard modules -->
-        <dependency>
-            <groupId>org.apache.eventmesh.dashboard.common</groupId>
-            <artifactId>eventmesh-dashboard-common</artifactId>
-            <version>0.0.1-SNAPSHOT</version>
-        </dependency>
         <dependency>
             <groupId>org.apache.eventmesh.dashboard.core</groupId>
             <artifactId>eventmesh-dashboard-core</artifactId>
             <version>0.0.1-SNAPSHOT</version>
         </dependency>
-        <dependency>
-            <groupId>org.apache.eventmesh.dashboard.service</groupId>
-            <artifactId>eventmesh-dashboard-service</artifactId>
-            <version>0.0.1-SNAPSHOT</version>
-        </dependency>
 
```

Comment:
Why these module deps are removed? 🤔

Reply:
The EventMesh dependency has been appended back, and the SpringBoot dependency is in the <!-- github laohu--> dependency

Reply:
The EventMesh dependency has been appended back, and the SpringBoot dependency is in the  <artifactId>dubbo-springmvc</artifactId>  dependency

Reply:
Better declare `dubbo-springmvc` has some important transitive dependencies.